foreman-selinux-disable

```diff
@@ -9,7 +9,7 @@ do
   if /usr/sbin/semodule -s $selinuxvariant -l >/dev/null; then
     # Remove all user defined ports (including the default one)
     /usr/sbin/semanage port -E | \
-      grep elasticsearch_port_t | \
+      grep -E '(elasticsearch|docker)_port_t' | \
```

Comment:
I don't understand you a single word here? And 42? No clue :-)

I mean we will have a *git conflict*. The extender grep will likely not be necessary.